beeflow/common/worker/worker.py

```diff
@@ -82,19 +82,24 @@ def prepare(self, task):
 
     def build_text(self, task):
         """Build text for task script; use template if it exists."""
-        # workflow_path = f'{self.workdir}/workflows/{task.workflow_id}/{task.name}-{task.id}'
         task_save_path = self.task_save_path(task)
         crt_res = self.crt.run_text(task)
         requirements = dict(task.requirements)
         hints = dict(task.hints)
+        main_command = crt_res.main_command[:]
+        # Append a tee to a file if requested
+        if task.stdout:
+            main_command.extend(['|', 'tee', task.stdout])
+        # TODO: It seems to me that there might be a better way to save stdout,
```

Comment:
I think we shouldn't have todo's in the source file, instead use issues. You can quote the code lines there.

Reply:
Hmmn, I think it's a good idea to have both issues and TODOs in the code. Perhaps we could reference the issue in the comment (something like `TODO:123: ...` for issue 123). Quoting code lines in the issue will quickly get out of date as code is refactored or the lines change as new code is added to the file. The TODO acts a marker in the code indicating exactly where something needs to be fixed/reviewed.